Adyen.Test/RecurringTest.cs

```diff
@@ -78,37 +78,17 @@ public void TestDisable()
         [TestMethod]
         public async Task TestDisableAsync()
         {
-            var client = base.CreateMockTestClientNullRequiredFieldsRequest("Mocks/recurring/disable-success.json");
+            var client = base.CreateMockTestClientApiKeyBasedRequestAsync("Mocks/recurring/disable-success.json");
             var recurring = new Service.Recurring(client);
             var disableRequest = this.CreateDisableRequest();
             var disableResult = await recurring.DisableAsync(disableRequest);
             Assert.AreEqual("[detail-successfully-disabled]", disableResult.Response);
         }
-
-        [TestMethod]
-        public void TestDisable803()

```

Comment:
Was this intended to be removed? What does this test do?

Reply:
is the 803 related to the http code `Incomplete HTTP response` ?

Reply:
Not sure tbh. Just think these old error tests do not really add anything anymore with the error handling we do in the client.